spec/ruby/core/string/shared/byte_index_ops.rb

```diff
@@ -0,0 +1,63 @@
+# -*- encoding: utf-8 -*-
+require_relative '../../../spec_helper'
+
+describe :byte_index_ops, shared: true do
```

Comment:
Well done.

suggestion(subjective):
```suggestion
describe :byte_index_common, shared: true do
```

Reply:
Looks better indeed, updating.